db/migrate/20170921115315_create_users.rb

```diff
@@ -1,15 +1,13 @@
 class CreateUsers < ActiveRecord::Migration[5.1]
```

Comment:
If this model is already merged everyone will need to redo this migration, which is not obvious.